client/components/Contact.scss

```diff
@@ -120,12 +120,66 @@
           width: 100%;
           border-radius: 4px;
           line-height: 1.56;
-          margin: 0px 0px 20px 0px;
+          margin: 0px 0px 8px 0px;
           border: solid 1px #c9c9c9;
           background-color: var(--white);
           padding: 16px 24px 12px 24px;
         }
 
+        &__checkbox {
+          display: block;
+          margin: 8px 0px;
+
+          input {
+            opacity: 0;
+          }
+
+          label {
+            position: relative;
+            padding-left: 22px;
+            font-family: Roboto;
+            font-size: 18px;
+            line-height: 1.56;
```

Comment:
Is the line-height necessary?

Reply:
Yes, this will make the parent block get the correct height.